packages/types/src/customize-symbol.test.ts

```diff
@@ -8,25 +8,27 @@ describe('Customizing metadata', () => {
       display:
         'delegation_penumbravalid1fjuj67ayaqueqxg03d65ps5aah6m39u39qeacu3zv2cw3dzxssyq3yrcez',
     });
-    customizeSymbol(metadata);
-    expect(metadata.symbol).toBe('Delegated UM (fjuj67ayaqueqxg03d65ps5aa...)');
+
+    expect(customizeSymbol(metadata).symbol).toBe('Delegated UM (fjuj67ayaqueqxg03d65ps5aa...)');
```

Comment:
`customizeSymbol` is now a pure function that returns a new (and modified) copy of the metadata